catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -64,6 +64,7 @@
         <bundle>mvn:ddf.catalog.core/catalog-core-resourcesizeplugin/${project.version}</bundle>
         <bundle>mvn:ddf.catalog.core/catalog-core-camelcomponent/${project.version}</bundle>
         <bundle>mvn:ddf.catalog.core/catalog-core-dynamic-registry/${project.version}</bundle>
+        <bundle>mvn:ddf.catalog.core/catalog-core-migratable/${project.version}</bundle>
```

Comment:
You need to separate out this bundle to its own feature as @coyotesqrl is suggesting and make catalog-core have a prereq on that feature. The feature that contains this bundle should also have a prereq on the platform feature `platform-migrate-all`